src/Page/Explore.elm

```diff
@@ -0,0 +1,311 @@
+module Page.Explore exposing
+    ( Model
+    , Msg(..)
+    , init
+    , subscriptions
+    , update
+    , view
+    )
+
+import Browser.Events
+import Browser.Navigation as Nav
+import Data.Country as Country exposing (Country)
+import Data.Dataset as Dataset exposing (Dataset)
+import Data.Food.Builder.Db as BuilderDb
+import Data.Food.Ingredient as Ingredient
+import Data.Impact as Impact
+import Data.Key as Key
+import Data.Scope as Scope exposing (Scope)
+import Data.Session exposing (Session)
+import Data.Textile.Db exposing (Db)
+import Data.Textile.Material as Material
+import Data.Textile.Product as Product
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Page.Explore.Countries as ExploreCountries
+import Page.Explore.FoodIngredients as FoodIngredients
+import Page.Explore.Impacts as ExploreImpacts
+import Page.Explore.Table as Table
+import Page.Explore.TextileMaterials as TextileMaterials
+import Page.Explore.TextileProducts as TextileProducts
+import Ports
+import RemoteData exposing (WebData)
+import Request.Food.BuilderDb as FoodRequestDb
+import Route
+import Views.Alert as Alert
+import Views.Container as Container
+import Views.Modal as ModalView
+
+
+type alias Model =
+    { dataset : Dataset
+    , scope : Scope
+    }
+
+
+type Msg
+    = NoOp
+    | CloseModal
+    | FoodDbLoaded (WebData BuilderDb.Db)
+
+
+init : Scope -> Dataset -> Session -> ( Model, Session, Cmd Msg )
+init scope dataset session =
+    ( { dataset = dataset, scope = scope }
+    , session
+    , Cmd.batch
+        [ if BuilderDb.isEmpty session.builderDb then
+            FoodRequestDb.loadDb session FoodDbLoaded
```

Comment:
Do we want to only load the food db if the scope is `Food`?

Reply:
done in https://github.com/MTES-MCT/ecobalyse/pull/208/commits/1a997da208ece37fed202c4e1655e5b2ff033e0f